src/components/JwtSection/JwtForm.jsx

```diff
@@ -171,6 +173,7 @@ function JwtForm({
 
             {/* Select */}
             <ExpirationSelect expiration={expiration} setExpiration={setExpiration} />
+            <TokenValidatior setTokenValidatior={setTokenValidatior} />
```

Comment:
It probably required Box wrapper to make a uniform offset 
![Screenshot from 2024-04-20 13-13-40](https://github.com/qdrant/qdrant-web-ui/assets/1935623/eebeaf50-53d0-49af-8a4a-4ab52bab45c2)